### ISSUE TITLE
quest deactivating as new quest added

### ISSUE THREAD

**Issue by Grogadog** [TheGameCreators/GameGuruRepo#4690]
quest deactivating as new quest added. The video is self explanatory

https://github.com/TheGameCreators/GameGuruRepo/assets/42348288/d08bf20c-df1d-4bc9-83d0-f8c2b087b1e8

Is this supposed to happen?

**Comment by Necrym59** [comment]
Yes  you can only have 1 quest actually activated at one time for the moment
